capability/capability.go

```diff
@@ -61,6 +63,11 @@ type Capabilities interface {
 	Apply(kind CapType) error
 }
 
+var (
+	ErrBoundingNotMine = errors.New("not support drop bounding cap of other process")
+	ErrAmbientNotMine  = errors.New("not support modify ambient cap of other process")
```

Comment:
Do we need these errors as part of a public API? If yes, why?

Reply:
Maybe someone use it to check the error type, like what to do in the test?
But maybe no, so I'll change it.

Reply:
> Maybe someone use it to check the error type, like what to do in the test?

For the test case, you can add `export_test.go` file having something like this:
```go
package capability

// Make some internal details available for tests.

var ErrBoundingNotMine = errBoundingNotMine
```